draft-ietf-quic-recovery.md

```diff
@@ -103,12 +103,12 @@ TCP implementations.
 
 Definitions of terms that are used in this document:
 
-Ack-eliciting Frames:
+Ack-eliciting frames:
```

Comment:
I'd either title-case or sentence-case these terms, but right now it's mixed.

Reply:
I think sentence-case is better.